server.js

```diff
@@ -15,10 +15,10 @@ app.locals.comments = [
 ]
 
 app.locals.favoriteMovies = [
-  {user_id: 59, movie_id: 1},
-  {user_id: 59, movie_id: 2},
-  {user_id: 59, movie_id: 3},
-  {user_id: 59, movie_id: 4},
+  {user_id: 59, movie_id: 475430},
+  {user_id: 59, movie_id: 338762},
+  {user_id: 59, movie_id: 508439},
+  {user_id: 59, movie_id: 603},
 ]
```

Comment:
This is great. Might go back and switch up some of the comment data so that it is more varied than all in one single movie.